specs/context/prismaContext.ts

```diff
@@ -0,0 +1,23 @@
+
+import { join } from "path";
+import { execSync } from "child_process";
+import {PrismaClient} from "@prisma/client";
+
+export function prismaTestContext() {
+  const prismaBinary = join(__dirname, "..", "..", "node_modules", ".bin", "prisma");
+  const prismaClient = new PrismaClient();
+  return {
+    async before() {
+      process.env.DATABASE_URL = 'your_test_database_url';
+      execSync(`${prismaBinary} migrate dev --name create_user_table`, { env: process.env });
```

Comment:
I think instead of overriding process.env.DATABASE_URL, you can create a new object here such as:
```suggestion
      execSync(`${prismaBinary} migrate dev --name create_user_table`, { env: {DATABASE_URL: process.env.NEW_ENV_VAR} });
```
And as Cris said you can declare that in your env file